test/unittests/mongoBackend/mongoNotifyContextAvailability_test.cpp

```diff
@@ -132,35 +186,86 @@ TEST(mongoNotifyContextAvailabilityRequest, ce1_En1_At0_Ok)
   /* Release connection */
   mongoDisconnect();
 
-  /* Delete mock */
-  delete timerMock;
-  delete notifierMock;
+  utExit();
 
 }
 
 /* ****************************************************************************
 *
-* ce1_En1nt_At0_Ok -
+* ce1_En1_At0_Ok_JSON -
 */
-TEST(mongoNotifyContextAvailabilityRequest, ce1_En1nt_At0_Ok)
+TEST(mongoNotifyContextAvailabilityRequest, ce1_En1_At0_Ok_JSON)
 {
   HttpStatusCode           ms;
   NotifyContextAvailabilityRequest   req;
   NotifyContextAvailabilityResponse  res;
 
-  /* Prepare mock */
-  NotifierMock* notifierMock = new NotifierMock();
-  EXPECT_CALL(*notifierMock, sendNotifyContextAvailabilityRequest(_,_,_,_))
-          .Times(0);
-  setNotifier(notifierMock);
+  utInit();
```

Comment:
I have take this opportunity to introduce the utInit/utExit in several files I touched that haven't them previously.

NTC